### ISSUE TITLE
ElementUI css-loader error

### ISSUE THREAD

**Issue by tangtan** [w1ndy/vue-webpack-ts-template#3]
I have met the same error as posted in [this issue](https://github.com/ElemeFE/element/issues/2541).
First, I install the element-ui package.
`npm i element-ui -S`
Then, I try to import the lib.
`
import 'element-ui/lib/theme-chalk/index.css'
`
And the error msg comes
```
error  in ./node_modules/element-ui/lib/theme-chalk/index.css
Module build failed (from ./node_modules/css-loader/index.js):
Unknown word (5:1)

  3 | // load the styles
  4 | var content = require("!!../../../css-loader/index.js??ref--13-1!./index.css");
> 5 | if(typeof content === 'string') content = [[module.id, content, '']];
    | ^
  6 | if(content.locals) module.exports = content.locals;
  7 | // add the styles to the DOM
  8 | var add = require("!../../../vue-style-loader/lib/addStylesClient.js").default


 @ ./node_modules/element-ui/lib/theme-chalk/index.css 4:14-152 14:3-18:5 15:22-160
 @ ./src/main.ts
 @ multi ./build/client-polyfill.js ./src/main.ts
```
I try to remove css-loader and change the below code.
```
export interface ICSSLoaders {
  // css: Loader | Loader[],
  postcss: Loader | Loader[],
  less: Loader | Loader[],
  sass: Loader | Loader[],
  scss: Loader | Loader[],
  stylus: Loader | Loader[],
  styl: Loader | Loader[],
  [key: string]: Loader | Loader[]
}
  return {
    // css: generateLoaders(),
    postcss: generateLoaders(),
    less: generateLoaders('less'),
    sass: generateLoaders('sass', { indentedSyntax: true }),
    scss: generateLoaders('sass'),
    stylus: generateLoaders('stylus'),
    styl: generateLoaders('stylus')
  }
```
But it pose another error msg.
```
ERROR in ./src/App.vue?vue&type=style&index=0&lang=css& (./node_modules/vue-loader/lib??vue-loader-options!./src/App.vue?vue&type=style&index=0&lang=css&) 20:0
Module parse failed: Unexpected character '#' (20:0)
You may need an appropriate loader to handle this file type.
| /* @import url('/node_module/element-ui/lib/theme-chalk/index.css'); */
| 
> #app {
|   font-family: 'Avenir', Helvetica, Arial, sans-serif;
|   -webkit-font-smoothing: antialiased;
 @ ./src/App.vue?vue&type=style&index=0&lang=css& 1:0-123 1:139-142 1:144-264 1:144-264
 @ ./src/App.vue
 @ ./src/main.ts
 @ multi ./build/client-polyfill.js ./src/main.ts
ERROR in ./src/components/HelloWorld.vue?vue&type=style&index=0&id=469af010&scoped=true&lang=css& (./node_modules/vue-loader/lib??vue-loader-options!./src/components/HelloWorld.vue?vue&type=style&index=0&id=469af010&scoped=true&lang=css&) 106:7
Module parse failed: Unexpected token (106:7)
You may need an appropriate loader to handle this file type.
| 
| 
> h1, h2 {
|   font-weight: normal;
| }
 @ ./src/components/HelloWorld.vue?vue&type=style&index=0&id=469af010&scoped=true&lang=css& 1:0-157 1:173-176 1:178-332 1:178-332
 @ ./src/components/HelloWorld.vue
 @ ./src/router/index.ts
 @ ./src/main.ts
 @ multi ./build/client-polyfill.js ./src/main.ts
```

**Comment by w1ndy** [comment]
Could you attach the steps to reproduce this error or a minimal
reproducible repo?
唐谈 <notifications@github.com>于2018年8月14日 周二下午7:21写道：

**Comment by tangtan** [comment]
I tried many methods, then I decide to link the css file to the index.html using CDN.
```
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width,initial-scale=1.0">
    <title>vue-webpack-ts-template</title>
    <link rel="stylesheet" href="https://unpkg.com/element-ui/lib/theme-chalk/index.css">
  </head>
```

**Comment by w1ndy** [comment]
This is caused by the duplicate css loaders due to the incorrect merging of webpack configurations. Fixed in  da04db2. Meanwhile I recommend importing css files inside <style> tags with ``@import "xxx.css"``, which results in a much cleaner code structure.
